torch/_dynamo/variables/tensor.py

```diff
@@ -1439,11 +1438,7 @@ def from_tensor_variable(cls, tensor_variable):
         return FakeItemVariable(**dict(tensor_variable.__dict__))
 
 
-class TensorSubclassVariable(VariableTracker):
-    def __init__(self, value, *args, **kwargs) -> None:
-        self.value = value
-        super().__init__(*args, **kwargs)
-
+class TensorSubclassVariable(UserDefinedClassVariable):
```

Comment:
This allows us to handle calls on the subclass type object, e.g., https://github.com/pytorch/pytorch/blob/10c4c45957546be5e1a86590834baaef68bba334/torch/nn/attention/bias.py#L281-L290